abdulfatai360/banka
# #165049405 Add User Account Creation Endpoint

#### What does this PR do?
Develop API endpoint for creating a new user account

#### Description of Task to be completed?
- Have `POST /api/v1/auth/signup` working
- Have JSON Web Token (JWT) generated for user on sign up

#### How should this be manually tested?
On your terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/18/head:ft-create-user-account-endpoint-165049405`
- Run `npm install` to install all project's dependencies onto your local machine
- Now that you have a copy of the branch, switch to it using  `git checkout ft-create-user-account-endpoint-165049405`
- Your directory will now be an exact copy of the PR.

On Postman:
- Test the endpoint

#### Any background context you want to provide?
When you want to test on Postman, ensure you fill in the user details as follows:
- firstName
- lastName
- otherName(_optional_)
- phone
- email
- password

#### What are the relevant pivotal tracker stories?
#165049405
